lib/money/bank/russian_central_bank.rb

```diff
@@ -58,7 +60,11 @@ def update_expired_at
       end
 
       def indirect_rate(from, to)
-        get_rate('RUB', to) / get_rate('RUB', from)
+        dividend = original_get_rate('RUB', to)
+        divider = original_get_rate('RUB', from)
+        return nil if dividend.nil? || divider.nil?
+
+        dividend.to_f / divider.to_f
```

Comment:
@olhor  Can we trim it down to this ?

```
def indirect_rate(from, to)
  get_rate('RUB', to) / get_rate('RUB', from)
  dividend = original_get_rate('RUB', to)
  divider = original_get_rate('RUB', from)
  
  dividend && divider && dividend.to_f / divider.to_f
end

Reply:
Line `get_rate('RUB', to) / get_rate('RUB', from)` will throw an exception if `get_rate` returns nil.

Reply:
@olhor I am sorry. I missed to remove that line - 

```
def indirect_rate(from, to)
  dividend = original_get_rate('RUB', to)
  divider = original_get_rate('RUB', from)
  
  dividend && divider && dividend.to_f / divider.to_f
end
```

Reply:
No problem for me.

Reply:
Is it possible that returned value can be `false`?